examples/cu_caterpillar/copperconfig.ron

```diff
@@ -113,4 +113,8 @@
         (src: "ct-6", dst: "gpio-6", msg: "cu_rp_gpio::RPGpioPayload"),
         (src: "ct-7", dst: "gpio-7", msg: "cu_rp_gpio::RPGpioPayload"),
     ],
+    monitor: (
```

Comment:
This section means that 'cargo run' actually runs the monitor in the terminal, it'd be worth noting that here

Maybe it'd be worth handling as a CLI param e.g. `cargo r -- --monitor` or the opposite for opting out like `cargo r -- --no-monitor` /  `cargo r -- --headless`

Reply:
Very good point but I think this is more the exception than the rule here as I am aiming at a good demo of this component. Normal monitors should be headless. 
What I am planning to do is to have some composable configs at some points where should be able to pick subsets of components dependendong on the specific robot you are deploying it on. I think we should put it in that logic, I am adding an issue so we can remember that.